server/plugin_release_test.go

```diff
@@ -366,24 +378,6 @@ func TestCreateTag(t *testing.T) {
 
 		gitMock.EXPECT().GetRefs(gomock.Eq(ctx), gomock.Eq(owner), gomock.Eq(repoName), gomock.Eq(fmt.Sprintf("tags/%s", tag))).Return(refs, nil, nil)
 
-		githubObj := &github.GitObject{
```

Comment:
These assertions were invalid as the tag was never expected to be created. I believe the dependencies updates for the underlying mocking library "fixed" this.